Sources/NIO/System.swift

```diff
@@ -438,7 +455,7 @@ internal enum Posix {
                     var off: off_t = offset
                     let result: ssize_t = Glibc.sendfile(descriptor, fd, &off, count)
                     if result >= 0 {
-                        written = result
+                        written = off_t(result)
```

Comment:
The only change in this pull that's not conditionally compiled only for Android, with the compiler complaining otherwise:
```
swift-nio/Sources/NIO/System.swift:458:35: error: cannot assign value of type 'ssize_t' (aka 'Int') to type 'off_t' (aka 'Int64')
```